### PR TITLE
Redesign of HXT Card

### PR DESCRIPTION
**Pull request by csnooke**
Jira Ticket [MUWM-5108](https://jira.cac.washington.edu/browse/MUWM-5108)

### CODE REVIEW

**Review by fanglinfang — APPROVED** [review]

File: myuw_vue/components/_common/husky-experience.vue [hunk 1]
```diff
@@ -6,40 +6,51 @@
...
-          <div class="position-absolute h-100 w-100 myuw-huskyexp-body">
-            <a
-              v-inner="articleTeaserTitle"
-              :title="`${articleTeaserTitle}. ${articleTeaserBody}`"
-              :href="expLink"
-              class="d-block h-100 px-3 py-4 pe-5"
-            >
+        </div>
+        <div class="col-10 col-lg-6">
               <h3 class="myuw-teaser-title">
```
Comment (fanglinfang) [inline myuw_vue/components/_common/husky-experience.vue:16]: Hi Cayla,  you may want to checkthe indentation (should be 2 spaces).

File: myuw_vue/components/_common/husky-experience.vue [hunk 2]
```diff
@@ -6,40 +6,51 @@
       </h2>
     </template>
     <template #card-body>
-      <div class="mx-n3 mb-n3 myuw-huskyexp">
-        <div class="position-relative overflow-hidden" style="max-height:300px;">
+      <div class="mb-2">
+                <div class="myuw-text-md me-auto">
```
Comment (fanglinfang) [inline myuw_vue/components/_common/husky-experience.vue:10]: Overly indented.

**Comment by fanglinfang** [comment]
William, please review the change in https://github.com/uw-it-aca/hx-toolkit/pull/32/files

File: myuw_vue/components/_common/husky-experience.vue [hunk 3]
```diff
@@ -6,30 +6,38 @@
       </h2>
     </template>
     <template #card-body>
-      <div class="mx-n3 mb-n3 myuw-huskyexp">
-        <div class="position-relative overflow-hidden" style="max-height:300px;">
+      <div class="mb-2">
+        <div class="myuw-text-md me-auto">
+        <div class="row">
```
Comment (fanglinfang) [inline myuw_vue/components/_common/husky-experience.vue:11]: No indentation on line 10, 11?

Comment (wmwash) [reply myuw_vue/components/_common/husky-experience.vue]: Good call, used code formatter on document.

**Review by wmwash — APPROVED** [review]

**Pull request merged**